protocols/luci-proto-batman-adv/htdocs/luci-static/resources/protocol/batadv_hardif.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+'require uci';
+'require form';
+'require network';
+
+network.registerPatternVirtual(/^bat.+$/);
+
+return network.registerProtocol('batadv_hardif', {
+	getI18n: function() {
+		return _('B.A.T.M.A.N. advanced Interface');
+	},
+	
+	getIfname: function() {
+		return this._ubus('l3_device') || this.sid;
+	},
+	
+	getOpkgPackage: function() {
+		return 'batman-adv';
+	},
+	
+	isFloating: function() {
+		return false;
+	},
+	
+	isVirtual: function() {
+		return false;
+	},
+	
+	getDevices: function() {
+		return null;
+	},
+	
+	containsDevice: function(ifname) {
+		return (network.getIfnameOf(ifname) == this.getIfname());
+	},
+	
+	renderFormOptions: function(s) {
+		var dev = this.getL3Device() || this.getDevice(), o;
+		
+		
+		// "master" is the batadv interface that we are linking to
```

Comment:
I don't think this comment is necessary. Can you please delete these.

Reply:
OK - fixed